src/index.html

```diff
@@ -22,6 +22,370 @@
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 Retina 5K"
+          class="card__image"
+        />
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rates">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rates-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 Retina 5K"
+          class="card__image"
+        />
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rates">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rates-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 Retina 5K"
+          class="card__image"
+        />
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rates">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rates-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
+      </div>
```

Comment:
You have multiple `card` blocks that repeat the same structure and content. To follow the DRY principle (Don't Repeat Yourself), you should consider using a template engine or server-side code to generate these blocks dynamically. If this is a static HTML exercise, please ignore this comment.